From ba735b4f0c27fbda5419f18060a91685472459ae Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Sun, 7 Mar 2021 02:33:36 +0100
Subject: [PATCH 11/16] most not do this, why?

---
 object.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/object.c b/object.c
index cb3c7ee7542..c47a21ae915 100644
--- a/object.c
+++ b/object.c
@@ -212,7 +212,9 @@ void *object_as_type(struct object *obj, enum object_type type, int quiet)
 	}
 	else {
 		if (!quiet)
-			object_is_type_or_error(obj, type);
+			error(_("object %s is a %s, not a %s"),
+			      oid_to_hex(&obj->oid),
+			      type_name(obj->type), type_name(type));
 
 		return NULL;
 	}
-- 
2.31.0.rc0.126.g04f22c5b82

