From 1f3a9ee185889bf41af8b081cad572cbc7f0e3aa Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Sun, 7 Mar 2021 02:33:49 +0100
Subject: [PATCH 12/16] can bring back this error handling, or useless?

---
 tag.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/tag.c b/tag.c
index 8b6d0639e12..7ca80eec3a7 100644
--- a/tag.c
+++ b/tag.c
@@ -227,11 +227,17 @@ int parse_tag(struct tag *item)
 	if (!data)
 		return error("Could not read %s",
 			     oid_to_hex(&item->object.oid));
+	/*if (type != OBJ_TAG) {
+		free(data);
+		return error("Object %s not a tag",
+			     oid_to_hex(&item->object.oid));
+	}*/
 	ret = object_is_type_or_error(&item->object, OBJ_TAG);
 	if (ret) {
 		free(data);
 		return ret;
 	}
+
 	ret = parse_tag_buffer(the_repository, item, data, size);
 	free(data);
 	return ret;
-- 
2.31.0.rc0.126.g04f22c5b82

