From f7e3a8013b1bc41a83e7d8899055cf80cc3f1f8d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Sun, 7 Mar 2021 14:04:50 +0100
Subject: [PATCH 14/16] add bug

---
 tag.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/tag.c b/tag.c
index 7ca80eec3a7..1757096f3ec 100644
--- a/tag.c
+++ b/tag.c
@@ -224,9 +224,13 @@ int parse_tag(struct tag *item)
 	if (item->object.parsed)
 		return 0;
 	data = read_object_file(&item->object.oid, &type, &size);
-	if (!data)
+	if (!data) {
+		BUG("noes");
 		return error("Could not read %s",
 			     oid_to_hex(&item->object.oid));
+	}
+	if (type != OBJ_TAG)
+		BUG("unreachable");
 	/*if (type != OBJ_TAG) {
 		free(data);
 		return error("Object %s not a tag",
-- 
2.31.0.rc0.126.g04f22c5b82

