From 0bc319d57a3a82b02cb018a89c219721e65d6d3c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Sun, 7 Mar 2021 14:13:55 +0100
Subject: [PATCH 15/16] debugging

---
 object-file.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/object-file.c b/object-file.c
index c28da6123cf..40e22478d06 100644
--- a/object-file.c
+++ b/object-file.c
@@ -1649,8 +1649,10 @@ void *read_object_file_extended(struct repository *r,
 		return data;
 
 	obj_read_lock();
-	if (errno && errno != ENOENT)
+	if (errno && errno != ENOENT) {
+		BUG("noes");
 		die_errno(_("failed to read object %s"), oid_to_hex(oid));
+	}
 
 	/* die if we replaced an object with one that does not exist */
 	if (repl != oid)
-- 
2.31.0.rc0.126.g04f22c5b82

