From 60f893bd9fe329bd5cf8ec513d10ec00e85feb2c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avar@cpan.org>
Date: Fri, 7 Mar 2008 16:06:31 +0000
Subject: [PATCH 6/6] exit after producing the error about anonymous having a password

---
 git-cvsserver.perl |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/git-cvsserver.perl b/git-cvsserver.perl
index aa0299e..da48ebd 100755
--- a/git-cvsserver.perl
+++ b/git-cvsserver.perl
@@ -160,6 +160,7 @@ if ($state->{method} eq 'pserver') {
         if (length($password) > 1 ) {
             print "E Don't supply a password for the `anonymous' user\n";
             print "I HATE YOU\n";
+            exit 1;
         }
 
         # Fall through to LOVE
-- 
1.7.0

