From 82d7458f22998c610f17338d730315399ff61db0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 30 Apr 2018 09:22:54 +0000
Subject: [PATCH 3/4] subtree: fix a test failure under GETTEXT_POISON
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/t7900-subtree.sh | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/t/t7900-subtree.sh b/t/t7900-subtree.sh
index eb223ff049..a6e7103f92 100755
--- a/t/t7900-subtree.sh
+++ b/t/t7900-subtree.sh
@@ -38,6 +38,16 @@ check_equal()
 	fi
 }
 
+check_equal_i18n()
+{
+	if test_have_prereq C_LOCALE_OUTPUT
+	then
+		check_equal "$@"
+	else
+		return 0
+	fi
+}
+
 undo()
 {
 	git reset --hard HEAD~
@@ -250,7 +260,7 @@ test_expect_success 'merge the added subproj again, should do nothing' '
 		# this shouldn not actually do anything, since FETCH_HEAD
 		# is already a parent
 		result=$(git merge -s ours -m "merge -s -ours" FETCH_HEAD) &&
-		check_equal "${result}" "Already up to date."
+		check_equal_i18n "${result}" "Already up to date."
 	)
 '
 
-- 
2.17.0.290.gded63e768a

